docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -168,6 +168,29 @@ paths:
           description: Profile information
           schema:
             $ref: '#/definitions/Profile'
+  /profile/groups:
+    get:
+      summary: Get list of groups for profile
+      description: >
+        Retrieve a list of accessible, applicable groups.
+
+        Note: This API service is pre-release and currently under development.
+        Be aware that parameters and response body will be changing.
+
+      parameters:
+        - name: authority
+          in: query
+          description: >
+            Used to identify available public groups.
+            Applies only to non-authorized requests.
+          required: false
+          type: string
+          default: "hypothes.is"
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/ProfileGroupResults'
```

Comment:
So right not `GET /profile/groups` takes only `?authority`, but other params (like `document_uri`) will be added later

Reply:
Correct. I didn't want anything in the rendered docs that was not implemented yet.